internal/provider/resource_aci_rest_managed.go

```diff
@@ -149,6 +151,12 @@ func (r *AciRestManagedResource) Schema(ctx context.Context, req resource.Schema
 				Default:             stringdefault.StaticString(globalAnnotation),
 				MarkdownDescription: `The annotation of the ACI object.`,
 			},
+			"escape_html": schema.BoolAttribute{
+				Optional:            true,
+				Computed:            true,
```

Comment:
I'm not sure if providing Computed: true is allowed or even makes sense here?
@lhercot @akinross your thoughts?

Reply:
This might be a new behaviour in the framework but in SDKv2 https://developer.hashicorp.com/terraform/plugin/sdkv2/schemas/schema-behaviors the description is different.
Good to know!!